pkg/cmd/roachtest/tests/tpcc.go

```diff
@@ -740,6 +740,25 @@ func registerTPCC(r registry.Registry) {
 		},
 	})
 
+	r.Add(registry.TestSpec{
+		Name:              "tpcc-nowait/nodes=5/w=1000",
```

Comment:
our test naming is a mess, in particular for anything related to tpcc. 
But for this new addition we should at least include the cpu count:

`tpcc-nowait/w=1000/nodes=5/cpu=16` (mimics tpccbench)
`fmt.Sprintf("tpcc-nowait/w=1000/%s", spec)` (mimics tpcc-headroom)

Reply:
Went with your first suggestion.  We should probably change the others in this file to match this convention, but I'm reluctant to do that now.  Doing so in a dedicated PR probably makes more sense.